rust/kcl-lib/src/std/axis_or_reference.rs

```diff
@@ -58,4 +58,24 @@ impl Axis3dOrPoint3d {
             Axis3dOrPoint3d::Point(point) => point.clone(),
         }
     }
+
+    pub fn to_origin(&self) -> [TyF64; 3] {
+        match self {
+            Axis3dOrPoint3d::Axis { origin, .. } => origin.clone(),
+            Axis3dOrPoint3d::Point(..) => [
```

Comment:
This is a confusing function, it might be unexpected that the point field (which you've hidden with `..`) becomes 0,0,0 instead of using the point field.

Maybe better as a function `fn axis_origin(&self) -> Option<[f64; 3]>` and then callers can use `.axis_origin.unwrap_or_default()`.